tag-publish-docker-image.sh

```diff
@@ -0,0 +1,93 @@
+#!/bin/bash
```

Comment:
I started playing around a little bit with this script. I don't understand what is happening when I don't check the code. Please add a lot more feedback to the scripts to explain where I am and what is happening.

Just some examples (they may be more things to add)
- was the docker login successful?
- What kind of tags will be created (release-tag? hash-tag?)
- Was the deployment successful?
- If nothing was executed, please give me some feedback also